tests/kuttl/common/scripts/validate_test_token.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+set -euxo pipefail
+
+seconds=1
+while [ $seconds -le 30 ]; do
+    rotatedat=$(oc get secret keystone -n $NAMESPACE -o jsonpath="{.metadata.annotations['keystone\.openstack\.org/rotatedat']}")
+    if [ $rotatedat != "2009-11-10T23:00:00Z" ]; then
+        break
+    fi
+    sleep 1
+    seconds=$(( $seconds + 1 ))
+done
+
```

Comment:
we have to wait for the keystoneapi to be restarted. otherwise we just test against the old deployment. when the annotation on the secret got updated it does not mean the deployment pod already got updated.
It could be a stupid sleep for maybe 20 or 30s, or probably better to exec into the keystone pod and check if it has the rotated keys on place.

Reply:
You're right. I'll opt for the sleep, since the mounted secret data can change without restarting the pod.

Reply:
I'll sleep for 20s and then do` oc rollout status deployment keystone -n openstack`